specification/purview/data-plane/Azure.Purview.Catalog/preview/2020-12-01-preview/purviewcatalog.json

```diff
@@ -0,0 +1,6748 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Purview Catalog Service REST API Document",
+    "description": "Purview Catalog Service is a fully managed cloud service whose users can discover the data sources they need and understand the data sources they find. At the same time, Data Catalog helps organizations get more value from their existing investments. This spec defines REST API of Purview Catalog Service.",
+    "version": "2020-12-01-preview"
+  },
+  "x-ms-parameterized-host": {
+    "hostTemplate": "{Endpoint}",
+    "useSchemePrefix": false,
+    "parameters": [
+      {
+        "$ref": "#/parameters/Endpoint"
+      }
+    ]
+  },
+  "basePath": "/api",
+  "schemes": [
+    "https"
+  ],
+  "tags": [
+    {
+      "name": "DiscoveryREST",
+      "description": "REST interface for data discovery using dsl or full text search."
+    },
+    {
+      "name": "EntityREST",
+      "description": "REST for a single entity."
+    },
+    {
+      "name": "GlossaryREST"
+    },
+    {
+      "name": "LineageREST",
+      "description": "REST interface for an entity's lineage information."
+    },
+    {
+      "name": "RelationshipREST",
+      "description": "REST interface for entity relationships."
+    },
+    {
+      "name": "TypesREST",
+      "description": "REST interface for CRUD operations on type definitions."
+    }
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "flow": "implicit",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      },
+      "type": "oauth2",
+      "description": "Azure Active Directory OAuth2 Flow."
+    }
+  },
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "paths": {
+    "/atlas/v2/entity": {
+      "post": {
+        "tags": [
+          "EntityREST"
```

Comment:
How are these tags used, does the "REST" suffix need to be removed here too?

Reply:
Tags are used for grouping operations. We keep "REST" in tags to be compatible with Atlas swagger.